docker/docker-compose.yml

```diff
@@ -19,6 +19,8 @@ services:
       - NODE_ENV
       - API_HOST
       - STATUS_PORT
+      - STATUS_URL
+      - POSTS_URL=http://posts:5555
```

Comment:
This belongs in the development docker-compose, not in the base one.  The only place we have the localhost issue is running locally.  Same idea as here https://github.com/Seneca-CDOT/telescope/blob/master/docker/development.yml#L107.

Reply:
Moved to development.yml.